Bust Config.rb

```diff
@@ -27,6 +27,8 @@
   "Z Andra emo" => nil, # To only allow face 4, see below
   "Z Andra emoN" => "AndraN", # post-debauchment
   "Z Orcent" => "Orcent1",
+  "Z Elven ambassador" => "Elven Ambassador",
+  "Z Zirantian ambassador" => "Uolian",
```

Comment:
There's no file named `Z Zirantian ambassador.png` in the PR at the moment.

Reply:
That's Uolian's existing faceset, which doesn't need edited to line up with the new bust.